refget.md

```diff
@@ -120,7 +120,7 @@ Content-type: text/vnd.ga4gh.refget.v1.0.0+plain
 
 | Parameter | Data Type | Required | Description                                                                                                                                                                                                         |
 |-----------|-----------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `id`      | string    | Yes      | A string specifying the sequence to be returned. The identifier shall be a checksum derived from the sequence using one of the supported checksum algorithms, or an alias for the sequence supported by the server. |
+| `id`      | string    | Yes      | A string specifying an identifier for the sequence to be returned. The identifier shall be a checksum derived from the sequence using one of the supported checksum algorithms, or a server specific checksum algorithm identifier. |
```

Comment:
Sounds a bit like `id` could be `FREDS_ALGORITHM` — i.e. an identifier for the algorithm. Perhaps something like _…using one of the defined checksum algorithms or a server-specific checksum algorithm…_?